loader.lua

```diff
@@ -117,7 +133,7 @@ local function spawnCard(face, position, flipped, onFullySpawned)
             obj.setDescription(face.oracleText)
             obj.setCustomObject({
                 face = face.imageURI,
-                back = "https://gamepedia.cursecdn.com/mtgsalvation_gamepedia/f/f8/Magic_card_back.jpg?version=0ddc8d41c3b69c2c3c4bb5d72669ffd7"
+                back = getCardBackInputValue()
```

Comment:
Maybe rename this to getCardBack now that we're not directly getting it from the input? Just a nitpick.
```suggestion
                back = getCardBack()
```

Reply:
Fixed